lib/CPathfinder.cpp

```diff
@@ -862,7 +862,7 @@ void DestinationActionRule::process(
 				else
 					action = CGPathNode::BLOCKING_VISIT;
 			}
-			else if(destination.isGuardianTile)
+			else if(destination.guarded)
```

Comment:
guarded will return true for neighbour tiles too?

Reply:
yes, as far as I understood. You can also see tested cases in #800